lib/glimesh_web/live/user_live/stream.ex

```diff
@@ -111,9 +111,10 @@ defmodule GlimeshWeb.UserLive.Stream do
     message =
       if lost_packets > 6,
         do:
-          gettext(
-            "We're detecting some networking problems between you and the streamer. You may experience video drops, jitter, or other issues!"
-          ),
+        gettext(
+          "We're detecting some networking problems between you and the streamer. You may experience video drops, jitter, or other issues! If this continues, the streamer is recommended to submit a ticket in #streaming-help in our"
+        ) <> " <a href=\"https://discord.gg/Glimesh\">Discord</a>."
+        |> Phoenix.HTML.raw(),
```

Comment:
Thanks for getting this done Mytho!

Since `gettext()` can be "user input" via our translation service, we don't want to call raw() with it or we could be injected. What I normally do is use UI patterns that split the link & the text. What we may want to do here is add an independent button for _all_ warnings we show inside the debug modal here: https://github.com/Glimesh/glimesh.tv/blob/dev/lib/glimesh_web/live/user_live/stream.html.leex#L105-L107

Something like:
> We're detecting some networking problems between you and the streamer. You may experience video drops, jitter, or other issues! If this continues, the streamer is recommended to submit a ticket in #streaming-help in our Discord
> [Join Our Discord!] (this would be a button)

Reply:
Ok, do you want it below the alert class or below?
![image](https://user-images.githubusercontent.com/58316242/124620541-27626d80-de3f-11eb-87ca-119cbf954ebd.png)


![image](https://user-images.githubusercontent.com/58316242/124620728-51b42b00-de3f-11eb-9a36-05f3ff817c39.png)

Reply:
The latter looks good to me. What do you think?

Reply:
I think either one is fine, but I'll go with the last one.